starter-code/views/celebrities/edit.hbs

```diff
@@ -0,0 +1,13 @@
+<h2>Editing {{celeb.name}}</h2>
+
+
+
+    <form action="/celebrities/{{celeb.id}}" method="POST">
+        <label>Name:</label>
+        <input type="text" name="name" placeholder="Name">
```

Comment:
Just one thing: in the edit view you should fill the input with the information of the objet to edit it. Use the `value={{mov.name}}` to do so... `<input type="text" name="title" placeholder="Sharknado" value={{mov.name}}>`